lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -746,21 +773,42 @@ def rotate_axes(xs, ys, zs, zdir):
         return ys, zs, xs
     elif zdir == '-x':
         return zs, xs, ys
-
     elif zdir == 'y':
         return zs, xs, ys
     elif zdir == '-y':
         return ys, zs, xs
-
     else:
         return xs, ys, zs
 
 
+def _rotate_axes_vec(xyz, zdir):
+    """
+    Reorder coordinates so that the axes are rotated with zdir along
+    the original z axis. Prepending the axis with a '-' does the
+    inverse transform, so zdir can be x, -x, y, -y, z or -z
+    """
+    if zdir == 'x':
+        return xyz[[1, 2, 0]]
+    elif zdir == '-x':
+        return xyz[[2, 0, 1]]
+
+    elif zdir == 'y':
+        return xyz[[2, 0, 1]]
+    elif zdir == '-y':
+        return xyz[[1, 2, 0]]
```

Comment:
Again, maybe `np.roll()` might be more efficient? Dunno.

Reply:
No longer applicable as the corresponding function was removed.